tests/python_frontend/identifiers_and_keywords_test.py

```diff
@@ -237,7 +237,7 @@ def test_keyword_lambda():
         keyword_lambda(A, B)
     except Exception as e:
         print(e)
-        return False
+        return True
```

Comment:
was this change intentional?

Reply:
Yes, as we do not support lambda functions in the context that one is used in that particular test. The test was erroneously returning `False` because I initially thought I could support them with type annotations. This failed and then I just forgot to change the test. This brings up another point, returning `True` or `False` from those tests does not trigger `SUCCESS` or `ERROR`. This should be fixed in another PR.